apps/explorer/src/components/OwnedObjects/OwnedObject.tsx

```diff
@@ -12,14 +13,17 @@ type OwnedObjectTypes = {
 };
 
 function OwnedObject({ obj }: OwnedObjectTypes): JSX.Element {
+    const video = useResolveVideo(obj);
     const displayMeta = getObjectDisplay(obj).data;
+
     return (
         <ObjectDetails
             variant="small"
             id={obj.data?.objectId}
             type={trimStdLibPrefix(parseObjectType(obj))}
             name={displayMeta?.name ?? displayMeta?.description}
             image={displayMeta?.image_url}
+            video={video}
```

Comment:
I saw the `desc` says the image_url must be present. Do we check for that anywhere? I only see the `image` being optional in `ObjectDetails`.

Reply:
Yeah this component is only rendered when there's an image url that's resolved, this is done at a higher level